jc/parsers/netstat_linux.py

```diff
@@ -31,10 +31,15 @@ def normalize_interface_headers(header):
 
 
 def parse_network(headers, entry):
+    LIST_OF_STATES = ["ESTABLISHED", "SYN_SENT", "SYN_RECV", "FIN_WAIT1", "FIN_WAIT2", "TIME_WAIT", "CLOSED", "CLOSE_WAIT", "LAST_ACK", "LISTEN", "CLOSING", "UNKNOWN"]
```

Comment:
Could you add "7" to this list? I haven't seen any other numbers in any other examples and this seems to do the right thing.